tsconfig.json

```diff
@@ -15,16 +15,21 @@
         "noUnusedParameters": true,
         "baseUrl": "./",
         "paths": {
+            "@hostapiv2": [
```

Comment:
I'm trying out using the [paths](https://www.typescriptlang.org/tsconfig#paths) feature for importing things from the api folder.

It's sorta quirky when doing the auto-imports quick fix, but we can always change everything to be path based later.

Reply:
Never mind, I can't get this to work properly